README.md

```diff
@@ -25,14 +25,14 @@ The app launches a window displaying a map.
 2. In the import wizard, choose _Maven > Existing Maven Projects_, then click _Next_.
 3. Select the java-maven-starter-project as the project root directory.
 4. Click _Finish_ to complete the import.
```

Comment:
Suggest inserting following new step between current 4 and 5 (for parity with the IntelliJ steps). If however, you think it is overkill, leave it out. Here's the text I propose:

5. Select  _Project > Properties_ . In _Java Build Path_--ensure that in Libraries tab, _Modulepath_ is set to JRE System Library (JavaSE-11). In _Java Compiler_--ensure that the _Use compliance from execution environment 'JavaSE-11' on the 'Java Build Path'_ checkbox is selected.